bounter/tests/cms/test_cms_init.py

```diff
@@ -106,6 +106,15 @@ def test_invalid_width(self):
         with self.assertRaises(ValueError):
             CountMinSketch(width=2 ** 22 - 1, depth=8)  # width must be a power of 2!
 
+    def test_largest_cms(self):
+        cms = None
+        try:
```

Comment:
No need to try/except here, if exception raised - test failed, that's expected (-> `assertIsInstance`  no needed too).